cpp/src/traversal/bfs.cu

```diff
@@ -468,13 +497,22 @@ namespace detail {
     //In that case, distances is a working data
     if (directed && !computeDistances)
       ALLOC_FREE_TRY(distances, nullptr);
+
+    // In that  case, previous_visited_bmap has been allocated
+    if (sp_counters) {
+      ALLOC_FREE_TRY(previous_visited_bmap, nullptr);
```

Comment:
Aren't we getting of ALLOC_TRY / ALLOC_FREE_TRY?  
Need to reconcile with @kaatish on status/plan regarding RMM changes for BC/BFS feature

Reply:
Updated in d5b9a32, BC now also uses rmm::device_vector, the clean function still exists as there is cub related allocation